llvm/docs/GlobalISel/GenericOpcode.rst

```diff
@@ -592,8 +592,8 @@ G_FLOG, G_FLOG2, G_FLOG10
 
 Calculate the base-e, base-2, or base-10 respectively.
```

Comment:
The comment is confusing.

Reply:
Thats  a comments for `G_FLOG, G_FLOG2, G_FLOG10` above. Thats not related to this pr.